models/Offering.js

```diff
@@ -1,24 +1,41 @@
 const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
-const OfferingSchema = Schema({
-  comment: {
+
+const Item = Schema({
```

Comment:
I'm fine with this, but just to check:

on FE, everything other than "images" is required for items. Should I remove that validation check and let it be optional? Or should I just leave it as it is? (I don't think BE needs to make them required)

Reply:
I think its fine to leave comments optional as not everyone might actually write something.

Reply:
It's true, but if the user doesn't write anything in the comments, FE still always returns `comment: ""`, so I think it's fine if Peng wants to keep it as required. (Either option works, I think)